### PR TITLE
LG/DCOS-44615 GCP-Fix broken links to installation methods, change pr…

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
…ocedures to use numbered steps, add advanced configuration info

## Description

https://jira.mesosphere.com/browse/DCOS-44615

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 1]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 </p>
 
+After you log in, the DC/OS dashboard is displayed.
+
 <p align=center>
 <img src="./images/install/dcos-ui.png" />
 </p>
 
-# Scaling Your Cluster
+For additional information about creating a cluster on GCP with more advanced configuration options, see [Advanced GCP cluster configuration options](#AdvancedGCP).
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Please replace with:

>The cluster configuration in this tutorial has been intentionally kept simple to get you started.  The Mesosphere Universal Installer supports a wide range of inputs/variables to add to your `main.tf` and enable you to customize your DC/OS cluster to your needs.  

>These inputs/variables include everything from specifying the operating system (CentOS, CoreOS, RHEL) to the garbage collection time of your DC/OS Cluster.  For details, see [Advanced GCP cluster configuration options](#AdvancedGCP).

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 2]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+
+# Scaling the cluster
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Scaling Your DC/OS Cluster

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 3]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 
-Username: `bootstrapuser`
+Username: `bootstrapuser`<br>
 Password: `deleteme`
 
-<p class="message--important"><strong>IMPORTANT: </strong>This should NOT be used in a production environment and you will need to generate a password hash.</p>
+<p class="message--important"><strong>IMPORTANT: </strong>You should NOT use the default credentials in a production environment. When you create or identify an administrative account for the production environment, you also need to generate a password hash for the account.</p>
 
-# Creating a Cluster
+# Creating a cluster
```
[the remark this answers is not in the document]
Comment (yishan-lin) [reply pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Creating a DC/OS Cluster

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 4]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+
 <p align=center>
 <img src="./images/install/dcos-ui.png" />
 </p>
 
-# Scaling Your Cluster
+For additional information about creating a cluster on GCP with more advanced configuration options, see [Advanced GCP cluster configuration options](#AdvancedGCP).
+
+# Scaling the cluster
 Terraform makes it easy to scale your cluster to add additional agents (public or private) once the initial cluster has been created. Use the instructions below.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: The Universal Installer makes it easy to scale your DC/OS cluster.  You can quickly add additional agent nodes, public or private, once the initial DC/OS cluster has been created with the instructions below.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 5]
```diff
@@ -70,298 +71,510 @@ us-west1
...
-  cluster_name        = "my-open-dcos"
-  ssh_public_key_file = "~/.ssh/id_rsa.pub"
+      num_masters        = "1"
+      num_private_agents = "3"
+      num_public_agents  = "1"
 
-  num_masters        = "1"
-  num_private_agents = "3"
-  num_public_agents  = "1"
+      dcos_version = "1.11.4"
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Please change this to `dcos_version = "1.11.7"`

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 6]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+    output "public-agents-loadbalancer" {
+      value = "${module.dcos.public-agents-loadbalancer}"
+    }
+    ```
 
-output "public-agents-loadbalancer" {
-  value = "${module.dcos.public-agents-loadbalancer}"
-}
-```
+2) Re-run the new execution plan with the changes you have made to the `main.tf`configuration file.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: 👍

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 7]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 
-2) The changes are made to `main.tf`, now re-run the new execution plan.  
+    ```bash
+    terraform plan -out=plan.out
+    ```
 
-```bash
-terraform plan -out=plan.out
-```
+    This step ensures that the state is stable and confirms that you can create the resources necessary to scale the private agents to the desired number. Executing the plan adds the following resources as a result of scaling up the cluster’s private agents:
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Revert to original copy please - 

>This step ensures that the state is stable and confirms that you will create the resources necessary to scale the private agents to the desired number.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 8]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+    - One instance resource
+    - Two null resources which handle the DC/OS installation and prerequisites in the background.
 
-This step ensures that the state is stable and to confirm that you will create the resources necessary to scale the private agents to the desired number.
+    You should see a message similar to the following:
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >After running `terraform plan`, you should see a message similar to the following.  There will be `three` resources added as a result of scaling up the cluster’s private agents (1 instance resource & 2 null resources which handle the DC/OS installation and prerequisites in the background).

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 9]
```diff
@@ -70,298 +71,510 @@ us-west1
...
-After executing the plan, you should see a message similar to above message. There will be `three` resources added as a result of scaling up the cluster’s private agents (1 instance resource & 2 null resources which handle the DC/OS installation and prerequisites in the background).
+    ```bash
+    terraform apply plan.out
+    ```
 
-3) Now, that the plan is set, get Terraform to build/deploy the new set of resources.
+    You should see an output similar to the following:
+  <p align=center><img src="./images/scale/terraform-apply.png" /></p>
+  
+4) Check your DC/OS cluster using the DC/OS UI to verify the additional agents have been added.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Check your DC/OS cluster via the UI to verify that the additional agent nodes have been successfully added.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 10]
```diff
@@ -70,298 +71,510 @@ us-west1
...
-```bash
-terraform apply plan.out
-```
+    You should see `four` total nodes connected. For example:
+  <p align=center><img src="./images/scale/node-count-4.png" /></p>
 
-<p align=center>
-<img src="./images/scale/terraform-apply.png" />
-</p>
+# Upgrading the cluster
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Upgrading Your DC/OS Cluster

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 11]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+Terraform also makes it easy to upgrade the DC/OS cluster to a newer version of DC/OS.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Mesosphere Universal Installer makes upgrading your DC/OS cluster to newer patches & major versions of DC/OS easy and effortless.  In this tutorial, we will upgrade our cluster from DC/OS 1.11.7 to 1.12.0.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 12]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 
-Once you see an output like the message above image, then check your DC/OS cluster to ensure the additional agents have been added.  
+Read more about the upgrade procedure that Terraform performs in the official [DC/OS Upgrade](https://docs.mesosphere.com/1.10/installing/production/upgrading/) documentation.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >You can read more about the upgrade procedure that the Universal Installer performs in our official [DC/OS Upgrade] documentation.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 13]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 
-Now, you should see `four` total nodes connected like below via the DC/OS UI.
+To perform an upgrade:
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: To upgrade your DC/OS cluster:

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 14]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 
-<p align=center>
-<img src="./images/scale/node-count-4.png" />
-</p>
+1) Open the `main.tf` configuration file.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Open your `main.tf` file.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 15]
```diff
@@ -70,298 +71,510 @@ us-west1
...
 
+2) Modify the current DC/OS Version (`dcos_version`) to `1.11.5` and set the `dcos_install_mode` parameter to `upgrade`.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Change the current DC/OS version (`dcos_version`) variable to `1.12.0` and set the `dcos_install_mode` variable to `upgrade`.

By default, the `dcos_install_mode` variable is set to `install` in order to enable creation and scalability of your initial DC/OS cluster.  In order to upgrade with the Universal Installer, you must explicitly set the `dcos_install_mode` varibale to `upgrade`.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 16]
```diff
@@ -70,298 +71,510 @@ us-west1
...
-data "http" "whatismyip" {
-  url = "http://whatismyip.akamai.com/"
-}
+      num_masters        = "1"
+      num_private_agents = "3"
+      num_public_agents  = "1"
 
-module "dcos" {
-  source = "dcos-terraform/dcos/gcp"
+      dcos_version = "1.11.5"
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Change this to `1.12.0`

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 17]
```diff
@@ -70,298 +71,510 @@ us-west1
...
-
-You should see an output like below.
+    ```bash
+    terraform apply plan.out
+    ```
 
-<p align=center>
-<img src="./images/upgrade/terraform-plan.png" />
-</p>
+5) Verify the cluster was upgraded using the DC/OS UI.
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: Once the `terraform apply` completes, verify that your DC/OS cluster was upgraded via the UI.

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 18]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+  <p align=center><img src="./images/upgrade/cluster-details-open.png" /></p>
 
+# Maintaining the cluster
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Maintaining Your DC/OS Cluster

File: pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md [hunk 19]
```diff
@@ -70,298 +71,510 @@ us-west1
...
+For instructions on how to maintain your cluster, follow the [maintenance](https://github.com/dcos-terraform/terraform-gcp-dcos/tree/master/docs/maintain) documentation.
 
-3) Apply the plan.
+# Deleting the cluster
```
Comment (yishan-lin) [inline pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md]: >Deleting Your DC/OS Cluster

File: pages/1.10/installing/index.md [hunk 20]
```diff
@@ -29,9 +29,9 @@ You can [evaluate](/1.10/installing/evaluation/) the installation process using
 These installation methods are officially supported by Mesosphere.  
 
 Any of the following methods can be used to install DC/OS:
-- [Provision DC/OS on Amazon Web Services(AWS)](/1.10/installing/evaluation/mesosphere-supported/aws/): Install your DC/OS cluster on Amazon Web Services (AWS) by using the DC/OS Terraform templates on AWS.
-- [Provision DC/OS on Azure](/1.10/installing/evaluation/mesosphere-supported/azure/): Install your DC/OS cluster on Azure by using the DC/OS Terraform templates on Azure.
-- [Provision DC/OS on Google Cloud Platform (GCP)](/1.10/installing/evaluation/mesosphere-supported/gcp/): Install your DC/OS cluster on GCP by using the DC/OS Terraform templates on GCP. 
+- [Provision DC/OS on Amazon Web Services (AWS)](/1.10/installing/evaluation/mesosphere-supported-methods/aws/): Install your DC/OS cluster on Amazon Web Services (AWS) by using the DC/OS Terraform templates on AWS.
```
[the remark this answers is not in the document]
Comment (lsgunnlsgunn) [reply pages/1.10/installing/index.md]: Fixed in PR
https://github.com/mesosphere/dcos-docs-site/pull/1756

File: pages/1.10/installing/index.md [hunk 21]
```diff
@@ -29,9 +29,9 @@ You can [evaluate](/1.10/installing/evaluation/) the installation process using
...
+- [Provision DC/OS on Azure](/1.10/installing/evaluation/mesosphere-supported-methods/azure/): Install your DC/OS cluster on Azure by using the DC/OS Terraform templates on Azure.
```
[the remark this answers is not in the document]
Comment (lsgunnlsgunn) [reply pages/1.10/installing/index.md]: Fixed in PR
https://github.com/mesosphere/dcos-docs-site/pull/1756

File: pages/1.12/installing/index.md [hunk 22]
```diff
@@ -29,7 +29,7 @@ You can [evaluate](/1.12/installing/evaluation/) the installation process using
 These installation methods are officially supported by Mesosphere.
 
 Any of the following methods can be used to install DC/OS:
-- [Provision DC/OS on Amazon Web Services(AWS)](/1.12/installing/evaluation/mesosphere-supported-methods/aws/): Install your DC/OS cluster on Amazon Web Services (AWS) by using the DC/OS Terraform templates on AWS.
+- [Provision DC/OS on Amazon Web Services (AWS)](/1.12/installing/evaluation/mesosphere-supported-methods/aws/): Install your DC/OS cluster on Amazon Web Services (AWS) by using the DC/OS Terraform templates on AWS.
```
[the remark this answers is not in the document]
Comment (lsgunnlsgunn) [reply pages/1.12/installing/index.md]: Fixed in PR
https://github.com/mesosphere/dcos-docs-site/pull/1756

File: pages/1.12/installing/index.md [hunk 23]
```diff
@@ -29,7 +29,7 @@ You can [evaluate](/1.12/installing/evaluation/) the installation process using
...
 - [Provision DC/OS on Azure](/1.12/installing/evaluation/mesosphere-supported-methods/azure/): Install your DC/OS cluster on Azure by using the DC/OS Terraform templates on Azure.
```
[the remark this answers is not in the document]
Comment (lsgunnlsgunn) [reply pages/1.12/installing/index.md]: Fixed in PR
https://github.com/mesosphere/dcos-docs-site/pull/1756

**Pull request merged**